ads/appnexus.js

```diff
@@ -105,17 +105,34 @@ function appnexusAst(global, data) {
     apntag = context.master.apntag;
 
     //preserve a global reference
+    /** @type {{showTag: function(string, Object)}} global.apntag */
     global.apntag = context.master.apntag;
   }
 
+  // check for ad responses received for a slot but before listeners are registered,
+  // for example when an above-the-fold ad is scrolled into view
   apntag.anq.push(() => {
-    apntag.onEvent('adAvailable', data.target, adObj => {
-      global.context.renderStart({width: adObj.width, height: adObj.height});
-      apntag.showTag(data.target, global.window);
-    });
+    const getAd = apntag.checkAdAvailable(data.target);
```

Comment:
Hi @protonate, `apntag.checkAdAvailable(data.target)` seems to return { getAd } object instead of a function.

Reply:
@liamqma Thanks for pointing this out, ast.js has been updated and this function should now be fixed